doc/DEVELOPER_TOOLING.md

```diff
@@ -1,9 +1,42 @@
-## Astyle
+## Code style (astyle)
 
-Automatic formatting is performed by astyle.  If you have make and astyle
-installed then this can be done with `make astyle`.
+Automatic formatting of source code is performed by [Artistic Style](http://astyle.sourceforge.net/).
 
-On Windows, there is an astyle extension for Visual Studio.
+If you have both `make` and `astyle` installed then this can be done with:
+
+```BASH
+make astyle
+```
+
+If you have only `astyle` then use:
+
+```BASH
+astyle --options=.astylerc --recursive src/*.cpp,*.h tests/*.cpp,*.h`
```

Comment:
There are also source files in tools/clang-tidy-plugin and tools/format

Reply:
Oh, I forgot they are getting formatted too.